examples/aws/storage-fsx/README.md

```diff
@@ -0,0 +1,163 @@
+# Deploying Kubeflow with Amazon FSx for Lustre as Persistent Storage
+
+This guide describes how to deploy Kubeflow on AWS EKS using Amazon FSx for Lustre as the Persistent Storage.
+
+## Prerequisites
+
+## 1. Prerequisites
+Follow the pre-requisites section from [this guide](../rds-s3/README.md#1-prerequisites) to:
+1. Install the CLI tools
+2. Clone the repo and checkout the right branch
+3. Create an EKS cluster and
+4. Make sure the following environment variables are set - 
+```
+export CLUSTER_NAME=<clustername>
+export CLUSTER_REGION=<clusterregion>
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+```
+
+## 1.0 Create the IAM Policy for the CSI Driver
+Create an IAM policy that allows the CSI driver's service account to make calls to AWS APIs on your behalf.
+
+a. Create the policy using the json file provided as follows - 
+```
+aws iam create-policy \
+    --policy-name Amazon_FSx_Lustre_CSI_Driver \
+    --policy-document file://examples/aws/storage-fsx/fsx-csi-driver.json
+```
+
+b. Create an IAM role and attach the IAM policy to it. Annotate the Kubernetes service account with the IAM role ARN and the IAM role with the Kubernetes service account name. You can create the role using eksctl as follows - 
+
+```
+eksctl create iamserviceaccount \
+    --name fsx-csi-controller-sa \
+    --namespace kube-system \
+    --cluster $CLUSTER_NAME \
+    --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/Amazon_FSx_Lustre_CSI_Driver \
+    --region $CLUSTER_REGION \
+    --approve \
+    --override-existing-serviceaccounts 
+```
+
+
+## 2.0 Configure the Service Account 
+
+a. In your AWS Console, go to the CloudFormation Service. Find the stack named `eksctl-prod-addon-iamserviceaccount-kube-system-fsx-csi-controller-sa` and check the outputs tab to find the name of the RoleARN that was created.
+
+b. Now, in the file `distributions/aws/aws-fsx-csi-driver/base/controller-serviceaccount.yaml`, edit line #9 to add the role created in the previous step. 
+
+
+## 3.0 Building manifests and deploying Kubeflow
+
+a. Deploy Kubeflow using the following single line installation command -
+
+```
+while ! kustomize build examples/aws/storage-fsx | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+```
+
+b. This should have installed the FSx CSI Driver into the default kube-system namespace for you. You can check using the following command - 
+```
+kubectl get csidriver -A
+
+NAME              ATTACHREQUIRED   PODINFOONMOUNT   MODES        AGE
+fsx.csi.aws.com   false            false            Persistent   5d17h
+```
+
+## 4.0 Create an Instance of the FSx Filesystem
```

Comment:
I would recomment to ommit further instruction and redirect user to follow AWS doc.